modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py

```diff
@@ -172,6 +172,7 @@ def init_persistent(self, node):
 
     def _prepare_configs(self, node):
         config = self.config.prepare_for_env(test_globals=self.globals, node=node, cluster=self)
```

Comment:
For now, this line useless.
We should have config as a parameter of the extend_config method.